ddsrouter_yaml/src/cpp/YamlReader.cpp

```diff
@@ -478,87 +476,84 @@ WildcardTopic YamlReader::get<WildcardTopic>(
 }
 
 template <>
-security::TlsConfiguration YamlReader::get<security::TlsConfiguration>(
+void YamlReader::fill(
```

Comment:
The client CA file should not be mandatory while the CA certificate of the server should not